_posts/release/2020-12-09-accumulo-1.10.1.md

```diff
@@ -9,8 +9,8 @@ draft: true
 
 Apache Accumulo 1.10.1 is a bug fix release of the 1.10 LTM release line.
 
-This release contains contributions from more than X contributors from the
-Apace Accumulo community in over X commits. These release notes are highlights of those changes. The full
+This release contains contributions from more than 6 contributors from the
+Apace Accumulo community in over 19 commits. These release notes are highlights of those changes. The full
```

Comment:
I think we started adding these numbers around 2.0.0 when the numbers were large and the degree of collaborative effort was impressive. When the numbers are low, I don't think it's necessary to include these. We can probably find some other way of wording it. That said, I don't care that much.

Reply:
I dropped the numbers - and accepted all of the other suggestions.